fastlane/lib/fastlane/actions/appaloosa.rb

```diff
@@ -104,6 +104,7 @@ def self.upload_on_appaloosa(api_key, store_id, binary_path, screenshots, group_
         req.body = { store_id: store_id,
                      api_key: api_key,
                      mobile_application_update: {
+                       description: description,
```

Comment:
you are right i guess, but all other field are optionals too, so it is why i do that.